### DISCUSSION TITLE
draw.io mathematical typeset enabled, renders in draw.io, but is blank/empty after saving and editing back into markdown

### DISCUSSION THREAD

**Discussion by jake-pfannenstiel** [requarks/wiki#6900] [Error / Bug Report]
I've never posted... I have a wiki.js page created with the markdown editor chosen. I click the button to insert a diagram from the left side of the page and the draw.io window automatically opens. I enable the Mathematical Typeset and create the text box.  Typing either '$$x^2$$' (MathJax format) and \`x^2`  (TeX format) shows the mathematical expression as desired while in draw.io. But after I Save & Exit, both the markdown preview and final page only display a blank diagram (or if \frac is used in the expression, then the only the line on the fraction is shown).

Technically this is over my head, but draw.io uses (I think) LaTeX or KaTeX instead of MathJax, like wiki.js markdown uses.  Even using AltCommands (&int;) only displays the actual text characters.

Is this a bug? I know I can export the draw.io diagram as an SVG file and import the SVG as an asset, but then it is not findable ( ctrl+f ) and I'd rather have the text selectable giving the user the ability to copy and paste if desired.

![image](https://github.com/requarks/wiki/assets/154618836/16b8955b-3d13-4c5a-a78e-213a70b574df)
Image above shows in draw.io diagram editor. Image below shows rendering of the image in the markdown preview pane.
![image](https://github.com/requarks/wiki/assets/154618836/0ae151e9-158b-48f3-bc88-f6be3ae0d9c9)

**Comment by ankit98verma** [comment]
> I've never posted... I have a wiki.js page created with the markdown editor chosen. I click the button to insert a diagram from the left side of the page and the draw.io window automatically opens. I enable the Mathematical Typeset and create the text box.  Typing either '$$x^2$$' (MathJax format) and \`x^2`  (TeX format) shows the mathematical expression as desired while in draw.io. But after I Save & Exit, both the markdown preview and final page only display a blank diagram (or if \frac is used in the expression, then the only the line on the fraction is shown).
> 
> Technically this is over my head, but draw.io uses (I think) LaTeX or KaTeX instead of MathJax, like wiki.js markdown uses.  Even using AltCommands (&int;) only displays the actual text characters.
> 
> Is this a bug? I know I can export the draw.io diagram as an SVG file and import the SVG as an asset, but then it is not findable ( ctrl+f ) and I'd rather have the text selectable giving the user the ability to copy and paste if desired.
> 
> 
> ![image](https://github.com/requarks/wiki/assets/154618836/16b8955b-3d13-4c5a-a78e-213a70b574df)
> Image above shows in draw.io diagram editor. Image below shows rendering of the image in the markdown preview pane.
> ![image](https://github.com/requarks/wiki/assets/154618836/0ae151e9-158b-48f3-bc88-f6be3ae0d9c9)
> 

Have you got any solution? I have same problem
